content/using_ec2_spot_instances_with_eks/deploy/tolerations_and_affinity.files/monte-carlo-pi-service-final.yml

```diff
@@ -27,21 +27,16 @@ spec:
       labels: 
         app: monte-carlo-pi-service 
     spec:
-      tolerations: 
```

Comment:
Tolerations is still in the release candidate. This would require testing with the latest eksctl version, which I can do with a follow up, once in the release candidate.